site/content/en/docs/concepts/workload_priority_class.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Workload Priority Class"
+date: 2023-10-02
+weight: 6
+description: >
+  A priorityClass which value is utilized by Kueue controller and is independent from pod's priority.
+---
+
+Sometimes developers would like to control workload's priority without affecting pod's priority, like these cases:
+
+- want to prioritize workloads that remain inactive for a specific duration
+- want to set a lower priority for development workloads and higher priority for production workloads
+
+A WorkloadPriorityClass is a priorityClass which value is utilized by Kueue controller and 
+is independent from pod's priority.
+
+A sample WorkloadPriorityClass looks like the following:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: WorkloadPriorityClass
+metadata:
+  name: sample-priority
+value: 10000
+description: "Sample priority"
+```
+
+## How to use WorkloadPriorityClass on Jobs
+
+The WorkloadPriorityClass is specified through a label `kueue.x-k8s.io/priority-class`.
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: sample-job
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+    kueue.x-k8s.io/priority-class: sample-priority
+spec:
+...
+```
+
+The following workload is generated by the yaml above. The `PriorityClassName` field can accept either `PriorityClass` or `WorkloadPriorityClass` name as a value. To distinguish, when using `WorkloadPriorityClass`, a `priorityClassSource` field has the `kueue.x-k8s.io/workloadpriorityclass` value. When using `PriorityClass`, a `priorityClassSource` field has the `scheduling.k8s.io/priorityclass` value.
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: Workload
+metadata:
+  name: job-sample-job-7f173
+spec:
+  priorityClassSource: kueue.x-k8s.io/workloadpriorityclass
+  priorityClassName: sample-priority
+  priority: 10000
+  queueName: user-queue
+...
+```
+
+For other job frameworks, you can set `WorkloadPriorityClass` through a label same for Job.
+The Following is an example of `MPIJob`.
+
+```yaml
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: pi
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+    kueue.x-k8s.io/priority-class: sample-priority
+spec:
+...
+```
+
+## The relationship between pod's priority and workload's priority
+
+There are three scenarios for creating a workload from a job.
```

Comment:
When creating a Workload for a given job, Kueue considers the following scenarios: